library/globals.inc.php

```diff
@@ -1421,18 +1426,58 @@ function gblTimeZones()
             'Lab Report',                     // default
             xl('Document category name for storage of electronically received MDM documents.')
         ),
+
         'generate_doc_thumb' => array(
             xl('Generate thumbnail'),
             'bool',
             '0',
             xl('Generate thumbnail images'),
         ),
+
         'thumb_doc_max_size' => array(
             xl('Thumbnail size'),
             'text',          // data type
             '100',           // default
             xl('Maximum size of thumbnail file')
         ),
+
+        'dicom_server_method' => array(
+            xl('DICOM Server'),
+            array(
+                '0' => xl('Orthanc'),
+            ),
+            '0',                              // default
+            xl('Option to retrieve DICOM images.')
+        ),
+
+        'dicom_host' => array(
+            xl('Orthanc HostName'),
+            'text',
+            'localhost',
+            xl('Orthanc host'),
+        ),
+
+        'orthanc_user' => array(
+            xl('Orthanc UserName'),
+            'text',
+            '',
+            xl('Username to connect to Orthanc'),
+        ),
+
+        'orthanc_pass' => array(
+            xl('Orthanc Password'),
+            'text',
+            '',
+            xl('Password to connect to Orthanc'),
+        ),
+
+        'orthanc_port' => array(
+            xl('Orthanc Port'),
+            'text',
+            '8042',
+            xl('Orthanc port'),
```

Comment:
Since these are straighforward, to make it easy on the translators, would make the 'descriptions' the same as the main labels.